### ISSUE TITLE
Cannonball impact logging seems broken (not written to files?)

### ISSUE THREAD

**Issue by iprice** [tgstation/tgstation#72336]
Reporting client version: 514.1589

## Round ID:
[197196](https://scrubby.melonmesa.com/round/197196)

## Testmerges:
- [Refactor air alarm TLV and modes.](https://www.github.com/tgstation/tgstation/pull/72315)
- [[NEW GHOST ANTAG] The Paradox Clone](https://www.github.com/tgstation/tgstation/pull/71141)

## Reproduction:

Was in the previous terry round that this mattered, but this (above) round ID covers the thunderdome testing for this report and has logs too

Spawn in a cannon (/obj/structure/cannon), one standard stack of cannonballs (/obj/item/stack/cannoball), gunpowder, bucket, torch (blah blah, set up a cannon with normal shot)
Spawn in a friend, ideally with a soul just to demonstrate its not ckey related
Shoot em

Note that in the player panel logs (in game) you will see e.g. this
```
[2022-12-28 21:43:37] RandomlyEvil/(Ivy Unlikely) (Thunderdome Arena (172,46,1)) (Event #1)
has been shot by the cannon with the cannonball
[2022-12-28 21:43:37] RandomlyEvil/(Ivy Unlikely) (Thunderdome Arena (172,46,1)) (Event #0)
suffered: Compound Fracture to right leg | Damage: 35 (rolled 133/145.109) | WB: -5
```

However if I scan (raw logs download here) attack.log then there's no "impact" logged
```
	Line  825: [2022-12-28 21:43:31.842] ATTACK: Iain0/(Bob Stange) transferred reagents (/datum/reagent/gunpowder (20u, 1 purity)) from the gunpowder barrel to [wooden bucket] (Thunderdome Arena (171,49,1))
	Line  832: [2022-12-28 21:43:36.182] ATTACK: Iain0/(Bob Stange) fired a cannon (Thunderdome Arena (172,49,1))
	Line  835: [2022-12-28 21:43:37.769] ATTACK: RandomlyEvil/(Ivy Unlikely) suffered: Compound Fracture to right leg | Damage: 35 (rolled 133/145.109) | WB: -5 (Thunderdome Arena (172,46,1))
```

similarly game.log only shows
```
	Line 2041: [2022-12-28 21:43:10.676] ADMIN: Iain0/(Bob Stange) spawned 1 x /obj/structure/fermenting_barrel/gunpowder at Thunderdome Arena (171,49,1)
	Line 2076: [2022-12-28 21:43:18.889] ADMIN: Iain0/(Bob Stange) spawned 1 x /obj/item/reagent_containers/cup/bucket/wooden at Thunderdome Arena (171,49,1)
	Line 2096: [2022-12-28 21:43:24.577] ADMIN: Iain0/(Bob Stange) spawned 1 x /obj/item/flashlight/flare/torch at Thunderdome Arena (172,49,1)
	Line 2150: [2022-12-28 21:43:36.182] GAME: Iain0/(Bob Stange) fired a cannon in Thunderdome Arena (172,48,1)
	Line 2165: [2022-12-28 21:43:40.046] SAY: RandomlyEvil/(Ivy Unlikely) "ow" (Thunderdome Arena (172,46,1))
	Line 2204: [2022-12-28 21:43:56.511] ADMIN: Iain0/(Bob Stange) checked the individual player panel for RandomlyEvil/(Ivy Unlikely) while in game.
	Line 2288: [2022-12-28 21:44:14.246] ADMIN: Iain0/(Bob Stange) spawned 1 x /obj/item/healthanalyzer/advanced at Thunderdome Arena (172,49,1)
```

There isn't an obvious way to tally up damage caused by a cannonball, you have to look for fractures (assuming it always causes one) and infer as no cause for the fracture is given.

Basically the question is why doesn't the "has been shot by the cannon with the cannonball" line seem to get to logfiles? Or am I looking in the wrong place (and if it is the wrong place, why?)

Thanks
